mypy/build.py

```diff
@@ -37,12 +37,13 @@
 from mypy import util
 from mypy.fixup import fixup_module_pass_one, fixup_module_pass_two
 from mypy.nodes import Expression
-from mypy.options import Options
 from mypy.parse import parse
 from mypy.stats import dump_type_stats
 from mypy.types import Type
 from mypy.version import __version__
 
+if TYPE_CHECKING:
```

Comment:
Did you see the comment on line 25 above? Please use MYPY instead. (It will somehow not fail in the tests but it will fail in real life when mypy is running on 3.5.1, because TYPE_CHECKING is not in that version's typing.py.)